### ISSUE TITLE
关于计算效率

### ISSUE THREAD

**Issue by OvOYu** [Onedean/EAC#1]
log/PEMS/eac-60/eac.log--这个文件中
2024-09-15 20:57:54,981 - [*] Year 2011 load from data/PEMS/FastData/2011.npz
2024-09-15 20:57:55,636 - [*] Year 2011 Dataset load!
2024-09-15 20:57:55,638 - Total Parameters: 7310
2024-09-15 20:57:55,638 - Trainable Parameters: 7310
2024-09-15 20:57:55,639 - [*] Year 2011 Training start
2024-09-15 20:57:56,398 - node number torch.Size([83840, 12])
2024-09-15 20:57:59,119 - epoch:0, training loss:10207.2653 validation loss:31.1622
2024-09-15 20:58:02,012 - epoch:1, training loss:1299.8966 validation loss:18.8867
2024-09-15 20:58:04,860 - epoch:2, training loss:829.9320 validation loss:17.8796
2024-09-15 20:58:07,815 - epoch:3, training loss:764.7204 validation loss:17.5585
根据您的训练日志，我发现每个epoch只要约3秒
2025-03-12 17:30:00,295 - [*] Year 2011 Dataset load!
2025-03-12 17:30:00,300 - Total Parameters: 7310
2025-03-12 17:30:00,300 - Trainable Parameters: 7310
2025-03-12 17:30:00,301 - [*] Year 2011 Training start
2025-03-12 17:34:32,754 - node number torch.Size([83840, 12])
2025-03-12 17:39:05,714 - epoch:0, training loss:10024.4411 validation loss:33.6857
2025-03-12 17:48:08,667 - epoch:1, training loss:1654.2076 validation loss:23.6726
这是我目前的日志，一轮起码4分钟以上，配置如下

![Image](https://github.com/user-attachments/assets/85bf779a-d05f-43e1-8786-bb7f1329f112)
已经将GPUid设为了0，莫非是因为CPU的区别

![Image](https://github.com/user-attachments/assets/04fe5f81-deaf-4c34-8f43-fb113a2698ee)
再次恭喜和感谢你们出色的工作!
